README.md

```diff
@@ -224,7 +222,6 @@ sourceSets {
     jvmMain {
         dependencies {
             implementation kotlin('stdlib-jdk8')
```

Comment:
Can JVM-only users still use `kotlinx-serialization-runtime` artifact or they have to migrate to `kotlinx-serialization-runtime-jvm`?

Reply:
Ok, judging by `publishPlatformArtifactsInRootModule` the answer is yes

Reply:
Yes, exactly